bluepymm/prepare_combos/prepare_emodel_dirs.py

```diff
@@ -146,6 +146,12 @@ def create_and_write_hoc_file(emodel, emodel_dir, hoc_dir, emodel_params,
     """
     setup = tools.load_module('setup', emodel_dir)
 
+    if not hasattr(setup, 'evaluator'):
+        # for python3, to be able to import the module
+        from importlib import __import__
+        __import__("setup")
```

Comment:
What was the exact problem here that is being solved?

Wouldn't it any be better to move this into the load_module method?

Reply:
yes, you are right, maybe better to put in load_module method. 
The problem was to be able to do setup.evaluator with python3.
In the singlecell repo, to be python3 compatible, @DrTaDa removed the import in the __init__.py, so it was complaining here that there was no evaluator attribute to setup. After a few tries, I could find this version that sort of works. Another issue I encountered was that due to the name setup, it seems it was importing all the setup.py it could find.